src/wallet/rpcwallet.cpp

```diff
@@ -2792,6 +2792,107 @@ static RPCHelpMan createwallet()
     };
 }
 
+static RPCHelpMan restorewalletfrombackup()
+{
+    return RPCHelpMan{
+        "restorewalletfrombackup",
+        "\nRestore and loads a wallet from backup.\n",
+        {
+            {"wallet_name", RPCArg::Type::STR, RPCArg::Optional::NO, "The name for the wallet. If this is a path, the wallet will be restored at the path location."},
```

Comment:
The wallet will not be restored at an arbitrary path as below we are combining this with `GetWalletDir()`. I don't think we should mention paths at all for the help for `wallet_name`

```suggestion
            {"wallet_name", RPCArg::Type::STR, RPCArg::Optional::NO, "The name that will be applied to the restored wallet"},
```

Reply:
Changed to `wallet_name` only.